js/packages/candy-machine-ui/src/Home.tsx

```diff
@@ -313,13 +380,25 @@ const Home = (props: HomeProps) => {
             open: true,
             message: 'Congratulations! Mint succeeded!',
             severity: 'success',
+            hideDuration: 7000,
           });
+          refreshCandyMachineState('processed');
+        } else if (status && !status.err) {
+          setAlertState({
+            open: true,
+            message:
+              'Mint likely failed! Anti-bot SOL 0.01 fee potentially charged! Check the explorer to confirm the mint failed and if so, make sure you are eligible to mint before trying again.',
```

Comment:
Should we give a link to the explorer when this happens?

Reply:
I was thinking about that. Not sure how easy it is to include a clickable link in the alert component though? If its possible to set the state as a jsx component then sure I think it would be neat!